### PR TITLE
set external url

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/giantswarm/issues/11524

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -110,6 +110,7 @@ func mainE(ctx context.Context) error {
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.CrtFile, "", "Certificate file path to use to authenticate with Kubernetes.")
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.KeyFile, "", "Key file path to use to authenticate with Kubernetes.")
 	daemonCommand.PersistentFlags().String(f.Service.Prometheus.BaseDomain, "", "Base domain to create Prometheus Ingress resources under.")
+	daemonCommand.PersistentFlags().String(f.Service.Prometheus.Address, "", "Address to access Prometheus UI.")
```
Comment (omegas27) [inline main.go:113]: What is the difference between address and host?

Comment (mig4) [reply main.go:113]: Or `address` and `baseDomain` (which was the original option we had for setting external URL with the original _Ingress_ resource)?

**Review by omegas27 — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (omegas27) [reply main.go:113]: Actually, we have both in draughtman CM, address is the same as the host but with https in front :D

Comment (TheoBrigitte) [reply main.go:113]: address has the protocol

```yaml
          Prometheus:
            Address: https://prometheus.g8s.gauss.eu-west-1.aws.gigantic.io
            Host: prometheus.g8s.gauss.eu-west-1.aws.gigantic.io
```

**Pull request merged**